src/sources/exec/mod.rs

```diff
@@ -385,7 +390,13 @@ async fn run_command(
             byte_size,
         });
 
+        let total_size: usize = events.iter().map(ByteSizeOf::size_of).sum();
```

Comment:
Would it make sense to implement `ByteSizeOf` for `SmallVec` in the core to avoid needing to do this expanded form here?

Reply:
I dropped the `SourceSender` so I no longer need it.